macros/WebAssemblySidebar.ejs

```diff
@@ -0,0 +1,54 @@
+<%
+var currentSection = $0;
+var locale = env.locale;
+var slug = env.slug;
+var baseURL = "/" + locale + "/docs/WebAssembly/";
+
+var text = mdn.localString({
+  'en-US': {
+    'WebAssembly_home_page' : 'WebAssembly home page',
+    'Tutorials' : 'Tutorials',
+      'WebAssembly_concepts' : 'WebAssembly concepts',
+      'Compiling_to_wasm' : 'Compiling from C/C++ to WebAssembly',
+      'JavaScript_API' : 'Using the WebAssembly JavaScript API',
+      'Text_format' : 'Understanding WebAssembly text format',
+      'Text_format_to_wasm' : 'Converting WebAssembly text format to wasm',
+      'Bytecode' : 'Fetching WebAssembly bytecode',
+      'Caching_modules' : 'Caching compiled WebAssembly modules',
+      'Exported_functions' : 'Exported WebAssembly functions',
+    'Object_reference' : 'Object reference'
+  }
+});
+%>
+
+<section class="Quick_links" id="Quick_Links">
+
+<ol>
+  <li data-default-state="open"><a href="<%=baseURL%>"><strong><%=text['WebAssembly_home_page']%></strong></a>
+  <li data-default-state="open"><a href="<%=baseURL%>"><%=text['Tutorials']%></a>
+    <ol>
+      <li><a href="<%=baseURL%>Concepts"><%=text['WebAssembly_concepts']%></a></li>
+      <li><a href="<%=baseURL%>C_to_wasm"><%=text['Compiling_to_wasm']%></a></li>
+      <li><a href="<%=baseURL%>Using_the_JavaScript_API"><%=text['JavaScript_API']%></a></li>
+      <li><a href="<%=baseURL%>Understanding_the_text_format"><%=text['Text_format']%></a></li>
+      <li><a href="<%=baseURL%>Text_format_to_wasm"><%=text['Text_format_to_wasm']%></a></li>
+      <li><a href="<%=baseURL%>Fetching_WebAssembly_bytecode"><%=text['Bytecode']%></a></li>
+      <li><a href="<%=baseURL%>Caching_modules"><%=text['Caching_modules']%></a></li>
+      <li><a href="<%=baseURL%>Exported_functions"><%=text['Exported_functions']%></a></li>
+    </ol>
+  </li>
+  <li data-default-state="open"><a href="<%=baseURL%>"><%=text['Object_reference']%></a>
+    <ol>
+      <li><%- template("jsxref", ["Global_objects/WebAssembly", "WebAssembly"]) %></li>
```

Comment:
This can just be the following:

```
      <li><%- template("jsxref", ["WebAssembly"]) %></li>
      <li><%- template("jsxref", ["WebAssembly.Module"]) %></li>
      <li><%- template("jsxref", ["WebAssembly.Instance"]) %></li>
      <li><%- template("jsxref", ["WebAssembly.Memory"]) %></li>
      <li><%- template("jsxref", ["WebAssembly.Table"]) %></li>
      <li><%- template("jsxref", ["WebAssembly.CompileError"]) %></li>
      <li><%- template("jsxref", ["WebAssembly.LinkError"]) %></li>
      <li><%- template("jsxref", ["WebAssembly.RuntimeError"]) %></li>
```
(leave out the constructor brackets, as the pages talk about the object in general with all its properties, methods, constructor)